packages/muze/src/canvas/component-wrapper-maker.js

```diff
@@ -76,17 +85,10 @@ const createHeaderWrapper = (headerType, layoutManager, renderDetails) => {
             classPrefix: layoutConfig.classPrefix,
             ...target,
             alignWith: `${ROW_MATRIX_INDEX[0]}-${COLUMN_MATRIX_INDEX[1]}`,
-            alignment: LAYOUT_ALIGN.LEFT,
+            alignment: headerAlignmentMap[LEFT],
             className: configType.className
         });
-
-        if (headerConfig.align === CENTER) {
-            headerConfig.alignment = null;
-            headerConfig.alignWith = null;
-        }
-        if (headerConfig.align === RIGHT) {
-            headerConfig.alignment = RIGHT;
-        }
+        headerConfig.alignment = headerAlignmentMap[headerConfig.align];
```

Comment:
ou dont need this line, can add it on line number 88 itself

Reply:
Updated